test/unit_test/passes/onnx/test_perf_tuning.py

```diff
@@ -5,26 +5,33 @@
 import logging
 import math
 import re
-from test.unit_test.utils import create_dataloader, get_onnx_model
+from test.unit_test.utils import get_onnx_model
 from unittest.mock import MagicMock, PropertyMock, patch
 
 import psutil
 import pytest
+from torch.utils.data import DataLoader
 
+from olive.data.component.dataset import DummyDataset
 from olive.evaluator.metric_result import flatten_metric_result
 from olive.evaluator.olive_evaluator import OliveEvaluator, OnnxEvaluator
 from olive.hardware.accelerator import DEFAULT_CPU_ACCELERATOR, DEFAULT_GPU_CUDA_ACCELERATOR, AcceleratorSpec, Device
 from olive.passes.olive_pass import create_pass_from_dict
 from olive.passes.onnx.perf_tuning import PERFTUNING_BASELINE, OrtPerfTuning, PerfTuningRunner, generate_test_name
 
 
+# TODO(shaahji): Remove this once perf_tuning pass supports DataConfig
+def _create_dummy_dataloader(data_dir, batch_size=1, **kwargs):
+    return DataLoader(DummyDataset([[1, 1]]), batch_size=batch_size)
```

Comment:
doesn't the shape `[1,1]` already include the batch dimension. Is the data from this dataloader going to have shape `[1,1,1]` then?

Reply:
There is some ambiguity between the way tests are implementing and what perf tuning/metrics expect as the data.  The implementation of `DummyDataset` hasn't changed in what it used to return (except for using `rand()` instead of `ones()`). I have a follow up PR to make both tests and perf_tuning work consistently with batching input.

Reply:
as long as the test passes, sounds good to me. Wasn't sure if it would pass since the shape `[1,1,1]` doesn't match the input shape of the model.

Reply:
All tests are passing. But, as you noticed, there are inconsistencies in inputs across tests and passes. Each test seems to be authored to work with specific data input, some accounting for batching and others don't.